src/main/java/com/tealium/Tealium.java

```diff
@@ -42,9 +45,12 @@
          *            Required. Tealium profile name.
          */
         public Builder(String account, String profile) {
-            if (Util.isEmpty(this.account = account) || Util.isEmpty(this.profile = profile)) {
+            if ((account == null) || (account.length() == 0) || (profile == null) || (profile.length() == 0)) {
```

Comment:
Any way to add the Util.isEmpty() method back in?? Looks like you check for null or length 0 for more than variable

Reply:
The Util class has actually been removed. The reason for this is all the logic that lived in there actually better belonged somewhere else, or not at all. In the case of the is empty method, it used to be called exactly twice, both times in this very same line of code. If a separate "isEmpty" method is used, a reader of this code would have to navigate elsewhere to understand what is going on. If the logic "isEmpty" was more complicated then it may make sense, but for something this basic that doesn't get reused anywhere else I thought it would be more readable to just inline it and not require maintainers to go through the hassle of locating the "isEmpty" logic just to find it doesn't really do much.

Reply:
It may be worth noting that nested functions would help in this scenario which java doesn't support. Java 8 does have lambdas, however we're still supporting Java 7 so it's not an option.